src/service/company_service.rs

```diff
@@ -311,7 +311,7 @@ impl CompanyServiceApi for CompanyService {
         let contacts = self.contact_store.get_map().await?;
```

Comment:
there are many errors in `company_service`, because `peer_id` is still used (probably from the code from the merge)

Reply:
yaah, coming from the merged